youtube_dl/extractor/lcp.py

```diff
@@ -0,0 +1,122 @@
+        'info_dict': {
+            'id': 'd56d03e9',
+            'url': 're:http://httpod.scdn.arkena.com/11970/d56d03e9_[0-9]+.mp4',
+            'ext': 'mp4',
+            'title': 'Schwartzenberg (PRG) préconise à François Hollande de participer à une primaire à gauche'
+        }
+    }, {
+        'url': 'http://www.lcp.fr/emissions/politique-matin/271085-politique-matin',
+        'md5': '6cea4f7d13810464ef8485a924fc3333',
+        'info_dict': {
+            'id': '327336',
+            'url': 're:http://httpod.scdn.arkena.com/11970/327336_[0-9]+.mp4',
+            'ext': 'mp4',
+            'title': 'Politique Matin - Politique matin'
+        }
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+
+        # Extract the required info of the media files gathered in a dictionary
+        media_files_info = self.__extract_from_webpage(display_id, webpage)
+        # Some web pages embed videos from other platforms like dailymotion, therefore we pass on these URLs
+        if media_files_info is None:
+            return self.url_result(url, 'Generic')
+
+        video_formats = self.__get_video_formats(media_files_info)
+        video_thumbnails = self.__get_thumbnails(media_files_info)
+
+        return {
+            'id': media_files_info['EntryName'],
+            'title': self._og_search_title(webpage),
+            'formats': video_formats,
+            'thumbnails': video_thumbnails
+        }
+
+    def __extract_from_webpage(self, display_id, webpage):
+        """Extracts the media info JSON object for the video for the provided web page."""
+        embed_url = self.__extract_embed_url(webpage)
+        embed_regex = r'(?:[a-zA-Z0-9]+\.)?lcp\.fr/embed/(?P<clip_id>[A-za-z0-9]+)/(?P<player_id>[A-za-z0-9]+)/(?P<skin_name>[^\/]+)'
+
+        clip_id = self._search_regex(embed_regex, embed_url, 'clip id', group='clip_id', fatal=False)
+        player_id = self._search_regex(embed_regex, embed_url, 'player id', group='player_id', fatal=False)
+        skin_name = self._search_regex(embed_regex, embed_url, 'skin name', group='skin_name', fatal=False)
+
+        # Check whether the matches failed, which might be when dealing with other players (e.g., dailymotion stream)
+        if (clip_id is None) or (player_id is None) or (skin_name is None):
+            return None
+
+        return self.__extract_from_player(display_id, clip_id, player_id, skin_name)
+
+    def __extract_embed_url(self, webpage):
+        return self._search_regex(
+            r'<iframe[^>]+src=(["\'])(?P<url>.+?)\1',
+            webpage, 'embed url', group='url')
+
+    def __extract_from_player(self, display_id, clip_id, player_id, skin_name):
+        """Extracts the JSON object containing the required media info from the embedded arkena player"""
+        arkena_url = 'http://play.arkena.com/config/avp/v1/player/media/{0}/{1}/{2}/?callbackMethod=?'.format(clip_id,
```

Comment:
As you wish. Note that looks like it would be impossible to craft a proper `_VALID_URL` URI template since embed URI is not domain specific (https://play.arkena.com/embed/avp/v2/player/media/b41dda37-d8e7-4d3f-b1b5-9a9db578bdfe/1/129411, http://play.lcp.fr/embed/327336/131064/darkmatter/0), so turns out it may only be available via shortcut like `arkena:clip_id:skin:player_id` or via base extractor implementation.